redaxo/src/core/lib/util/timer.php

```diff
@@ -44,8 +44,12 @@ public function __construct($start = null)
      *
      * On sufficient user permissions - or in debug mode - this timings will be sent over the wire to the browser via server timing api http headers.
      *
+     * @psalm-template T
```

Comment:
Gibts nen cs fixer der die Reihenfolge der phpdocs sicherstellt? Wäre schön wenn es irgendwie konsistent wäre :-)

Reply:
Nur für die normalen, und zwar erst `@param`, dann `@throws`, dann `@return`.
Aber für die psalm-Varianten und Zusatzdinger gibt es da nix.

Reply:
Wobei wir von meiner Seite aus auch gerne jetzt in den normalen docs die spezielleren Typen nehmen können, wo es phpstorm kann.

Reply:
> normalen docs die spezielleren Typen nehmen können, wo es phpstorm kann

Kann man mit nem separaten PR machen